Libraries/ext/Headers/ext/optional

```diff
@@ -168,6 +168,14 @@ namespace std { namespace experimental { inline namespace fundamentals_v1 {
 #endif
 #endif
 
+#if defined(__clang__) || defined(__GNUC__)
```

Comment:
Ya, I'd use that if it was standard. Otherwise it only helps for Clang which has supported these since before it had C++ support... so can't be bothered.